app/assets/javascripts/listings/ListingController.js.coffee

```diff
@@ -177,20 +177,14 @@ ListingController = (
   $scope.sortedInformationSessions = ->
     ListingService.sortByDate($scope.listing.Information_Sessions)
 
+  $scope.getOrdinal = (n) ->
```

Comment:
feel like this should be a filter https://github.com/jdpedrie/angularjs-ordinal-filter than a Controller function